packages/gamut/src/Form/Input.tsx

```diff
@@ -117,7 +117,11 @@ export const Input = forwardRef<HTMLInputElement, InputWrapperProps>(
     const ShownIcon = Icon || icon;
 
     return (
-      <Box position="relative" textColor={color}>
+      <Box
+        display={rest?.type === 'hidden' ? 'none' : undefined}
```

Comment:
`rest` will always exist:

```suggestion
        display={rest.type === 'hidden' ? 'none' : undefined}
```